src/components/GitHubLinkExtension.jsx

```diff
@@ -0,0 +1,35 @@
+/* eslint import/no-extraneous-dependencies: ["error", {"devDependencies": true}] */
```

Comment:
We should remove this linter comment. You were exporting the GitHubLinkExtension above in index.js so all our dependencies should be defined